### PR TITLE
.Net Connector - 8.0 Release branch

### PR DESCRIPTION
**Pull request by nidhibhammar**
## What Changed?

Addresses [EC-3066](https://enterprisedb.atlassian.net/browse/EC-3066)

[EC-3066]: https://enterprisedb.atlassian.net/browse/EC-3066?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: product_docs/docs/net_connector/8.0/03_the_advanced_server_net_connector_overview.mdx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+---
+
+<div id="the_advanced_server_net_connector_overview" class="registered_link"></div>
+
+EDB .NET Connector is a .NET data provider that allows a client application to connect to a database stored on an EDB Postgres Advanced Server host. The .NET Connector accesses the data directly, allowing the client application optimal performance, a broad spectrum of functionality, and access to EDB Postgres Advanced Server features.
+
+The .NET Connector supports the following frameworks:
+
+-   .NET 7.0
```
Comment (xfischer) [inline product_docs/docs/net_connector/8.0/03_the_advanced_server_net_connector_overview.mdx:12]: Can you also add .NET 8.0

File: product_docs/docs/net_connector/8.0/04_installing_and_configuring_the_net_connector.mdx
Comment (moazzumali) [inline product_docs/docs/net_connector/8.0/04_installing_and_configuring_the_net_connector.mdx:1]: Please remove the s from this statement "The following sections describe these methods in details."

**Review by moazzumali — APPROVED** [review]

**Review by djw-m — APPROVED** [review]
LGTM

**Review by ericmack — COMMENTED** [review]
approved

**Pull request merged**